docs/config/intro.md

```diff
@@ -0,0 +1,59 @@
+Configuration Files
+===================
+
+Forge has built in support for creating and managing mod configuration files. The purpose of a configuration file is to add flexibility and allow end users to customize their experience.
+
+Creating
+--------
+
+Create a dedicated class. Use the builder instance to define the configuration structure and data.
+
+```java
+private BooleanValue mySetting;
+
+public ServerConfig(ForgeConfigSpec.Builder builder) {
+  builder.comment("examplemod settings"); // Comment at the start of the file
+  
+  builder.push("Example Category"); // 'Enter' the Category
```

Comment:
Having two comments saying 'Enter' the category and 'Exit' the category could better be replaced with one comment explaining how push/pull creates categories. It's not super clear what 'entering' a category means.

Reply:
Yes of course, but these comments feel like you didn't want to not explain anything about the push/pop but now the comments aren't very useful. I'd say to replace the `// 'Enter' the category` and `// 'Exit' the category` comments with one `// Use the push/pop methods to create categories` comment. But this isn't super important.